include/etiss/ETISS.h

```diff
@@ -66,6 +66,9 @@
 
 #include "etiss/LibraryInterface.h"
 #include "etiss/IntegratedLibrary/SimpleIni/SimpleIni.h"
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
+#include <boost/program_options/variables_map.hpp>
```

Comment:
I would suggest to move these into a ETISS.cpp because they are not needed by this header